model/model.wl

```diff
@@ -99,24 +99,28 @@ BetaMeanSig[mu_,sig_]:=BetaDistribution[(mu^2-mu^3-mu sig)/sig,((-1+mu) (-mu+mu^
 PosNormal[mu_,sig_]:=TruncatedDistribution[{0,\[Infinity]},NormalDistribution[mu,sig]]
 
 (* a function to generate the monte carlo simulations from combo of fit / assumed parameters *)
-generateSimulations[numberOfSimulations_, fitParams_, standardErrors_]:=<|
-"daysFromInfectedToInfectious"->RandomVariate[PosNormal[daysFromInfectedToInfectious0,1]],
-"daysUntilNotInfectiousOrHospitalized"-> RandomVariate[PosNormal[daysUntilNotInfectiousOrHospitalized0,0.5]],
-"daysToLeaveHosptialNonCritical"-> RandomVariate[PosNormal[daysToLeaveHosptialNonCritical0,2]],
-"daysTogoToCriticalCare"-> RandomVariate[PosNormal[daysTogoToCriticalCare0,1]],
-"daysFromCriticalToRecoveredOrDeceased"-> RandomVariate[PosNormal[daysFromCriticalToRecoveredOrDeceased0,1]],
-"pPCRH"->RandomVariate[NormalDistribution[pPCRH0,0.05]],
-"pPCRNH"->RandomVariate[NormalDistribution[pPCRNH0,0.02]],
-"daysForHospitalsToReportCases"->RandomVariate[PosNormal[daysForHospitalsToReportCases0,0.3]],
-"daysToGetTestedIfNotHospitalized"->RandomVariate[PosNormal[daysToGetTestedIfNotHospitalized0,1]],
-"daysToGetTestedIfHospitalized"->RandomVariate[PosNormal[daysToGetTestedIfHospitalized0,0.3]],
-"icuOverloadDeathPenalty0"->RandomVariate[PosNormal[icuOverloadDeathPenalty0,0.5]],
-"initialInfectionImpulse"->RandomVariate[PosNormal[initialInfectionImpulse0,3]],
-"importlength"->RandomVariate[PosNormal[importlength0,1]],
-"fractionOfCriticalDeceased"->RandomVariate[BetaMeanSig[fractionOfCriticalDeceased0,0.02]],
-"r0natural"->RandomVariate[PosNormal[fitParams["r0natural"],standardErrors["r0natural"]]],
-"importtime"->RandomVariate[PosNormal[fitParams["importtime"],standardErrors["importtime"]]]
-|>&/@Range[numberOfSimulations];
+generateSimulations[numberOfSimulations_, fitParams_, standardErrors_, cutoff_, stateParams_]:=Module[{},
+  {
+   RandomVariate[PosNormal[fitParams["r0natural"],standardErrors["r0natural"]]],
+ RandomVariate[PosNormal[daysUntilNotInfectiousOrHospitalized0,0.5]],
+ RandomVariate[PosNormal[daysFromInfectedToInfectious0,1]],
+ RandomVariate[PosNormal[daysToLeaveHosptialNonCritical0,2]],
+RandomVariate[PosNormal[pPCRNH0,0.02]],
+RandomVariate[PosNormal[pPCRH0,0.05]],
+ RandomVariate[PosNormal[daysTogoToCriticalCare0,1]], 
+ RandomVariate[PosNormal[daysFromCriticalToRecoveredOrDeceased0,1]],
+RandomVariate[BetaMeanSig[fractionOfCriticalDeceased0,0.02]],
+RandomVariate[PosNormal[fitParams["importtime"],standardErrors["importtime"]]], 
+RandomVariate[PosNormal[importlength0,1]],
+RandomVariate[PosNormal[initialInfectionImpulse0,3]],
+cutoff,
+stateParams["params"]["pS"],
+stateParams["params"]["pH"],
+stateParams["params"]["pC"],
+containmentThresholdRatio0,
+stateParams["icuCapacity"],
+stateParams["hospitalCapacity"]
+}&/@Range[numberOfSimulations]]
```

Comment:
Why the switch to an array away from an association?

Reply:
This had to do with how the PDE wants arguments to be applied (at least how rob had written it). I think there is some way I could get the values out of the association inline, but decided not to mess with it for now